packages/manager/apps/vrack-services/src/components/OrderSubmitModalContent.component.tsx

```diff
@@ -151,13 +132,13 @@ export const OrderSubmitModalContent: React.FC<OrderSubmitModalContentProps> = (
         variant={ODS_BUTTON_VARIANT.flat}
         disabled={!isContractAccepted || undefined}
         color={ODS_THEME_COLOR_INTENT.primary}
-        {...handleClick(() => {
+        {...handleClick(async () => {
```

Comment:
```suggestion
        {...handleClick(() => {
```

Since you don't `await`